skyportal/tests/frontend/test_scanning_page.py

```diff
@@ -0,0 +1,229 @@
+    manage_groups_token,
+):
+    candidate_id = str(uuid.uuid4())
+    for i in range(5):
+        status, data = api(
+            "POST",
+            "candidates",
+            data={
+                "id": f"{candidate_id}_{i}",
+                "ra": 234.22,
+                "dec": -22.33,
+                "redshift": 3,
+                "altdata": {"simbad": {"class": "RRLyr"}},
+                "transient": False,
+                "ra_dis": 2.3,
+                "filter_ids": [public_filter.id],
+            },
+            token=upload_data_token,
+        )
+        assert status == 200
+        assert data["data"]["id"] == f"{candidate_id}_{i}"
+
+    status, data = api(
+        "POST",
+        "groups",
+        data={"name": str(uuid.uuid4()), "group_admins": [user.username]},
+        token=manage_groups_token,
+    )
+    assert status == 200
+
+    driver.get(f"/become_user/{user.id}")
+    driver.get("/candidates")
+    for i in range(5):
+        driver.wait_for_xpath(f'//a[text()="{candidate_id}_{i}"]')
+    group_checkbox = driver.wait_for_xpath(
+        f'//input[starts-with(@name,"groupIDs[0]")]'
+    )
+    driver.scroll_to_element_and_click(group_checkbox)
+    submit_button = driver.wait_for_xpath('//span[text()="Submit"]')
+    driver.scroll_to_element_and_click(submit_button)
+    for i in range(5):
+        driver.wait_for_xpath_to_disappear(f'//a[text()="{candidate_id}_{i}"]')
+    driver.scroll_to_element_and_click(group_checkbox)
+    driver.scroll_to_element_and_click(submit_button)
+    for i in range(5):
+        driver.wait_for_xpath(f'//a[text()="{candidate_id}_{i}"]')
+
+
+def test_candidate_unsaved_only_filtering(
+    driver,
+    user,
+    public_candidate,
+    public_filter,
+    public_group,
+    upload_data_token,
+    manage_groups_token,
+):
+    candidate_id = str(uuid.uuid4())
+    for i in range(5):
+        status, data = api(
+            "POST",
+            "sources",
+            data={
+                "id": f"{candidate_id}_{i}",
+                "ra": 234.22,
+                "dec": -22.33,
+                "redshift": 3,
+                "altdata": {"simbad": {"class": "RRLyr"}},
+                "transient": False,
+                "ra_dis": 2.3,
+                "group_ids": [public_group.id],
+            },
+            token=upload_data_token,
+        )
+        assert status == 200
+        status, data = api(
+            "POST",
+            "candidates",
+            data={
+                "id": f"{candidate_id}_{i}",
+                "ra": 234.22,
+                "dec": -22.33,
+                "redshift": 3,
+                "altdata": {"simbad": {"class": "RRLyr"}},
+                "transient": False,
+                "ra_dis": 2.3,
+                "filter_ids": [public_filter.id],
+            },
+            token=upload_data_token,
+        )
+        assert status == 200
+        assert data["data"]["id"] == f"{candidate_id}_{i}"
+
+    status, data = api(
```

Comment:
Is this group being used anywhere?

Reply:
Good catch -- removed in https://github.com/skyportal/skyportal/pull/470/commits/21cb443d48b9f427ab7e3d45a674dbac8f25b837